packages/pipeline/src/pdk-pipeline.ts

```diff
@@ -315,7 +315,7 @@ export class PDKPipeline extends Construct {
               BRANCH: branch,
             }
           : undefined,
-      installCommands: ["npm install -g aws-cdk", "npx projen install"],
+      installCommands: ["npm install -g aws-cdk pnpm", "npx projen install"],
```

Comment:
Thanks for this! Users can override the install/build commands themselves to achieve this result but I think it makes sense to have the default commands work regardless of your package manager (or language).

I think it'd be good to additionally install `@aws/pdk` and then we can replace the `npx projen install` and `npx projen build` commands with `pdk install` and `pdk build` :)

Also, it looks like the build is failing due to the snapshots - please could you build locally and include the updated snapshots in your PR? Thanks!